includes/class-wc-beta-tester-admin-menus.php

```diff
@@ -184,10 +188,7 @@ protected function construct_ssr() {
 			$ssr = '';
 			foreach ( $response['environment'] as $key => $value ) {
 				$index = $key;
-				// @todo remove this hack after fix schema in WooCommerce, and Claudio never let you folks forget that need to write schema first, and review after every change, schema is the most important part of the REST API.
```

Comment:
Good old times ahahah

Reply:
That comment is almost worth leaving :).